### PR TITLE
feat(devtools): support vue-devtools

### PR DESCRIPTION
**Pull request by shenchaoran**
Before submitting a new pull request, please make sure:

- [ ] Test cases have been added/updated/passed for the code you will submit.
- [x] Documentation has added or updated.
- [x] Commit message is following the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters.
- [x] Squash the repeat code commits, short patches are welcome.

### CODE REVIEW

File: packages/hippy-vue/src/runtime/index.js [hunk 1]
```diff
@@ -253,4 +254,40 @@ Vue.getApp = getApp;
 // Register the built-in elements
 Vue.use(registerBuiltinElements);
 
-export default Vue;
+// vue devtools global hook
+const devtools = global.__VUE_DEVTOOLS_GLOBAL_HOOK__;
+setTimeout(() => {
```
Comment (zoomchan-cxj) [inline packages/hippy-vue/src/runtime/index.js:259]: 为什么要加setTimeout

File: packages/hippy-vue/src/runtime/index.js [hunk 2]
```diff
@@ -253,4 +254,40 @@ Vue.getApp = getApp;
...
+  if (config.devtools) {
+    if (devtools) {
+      devtools.emit('init', Vue);
+    } else if (
+      process.env.NODE_ENV !== 'production'
+      && process.env.NODE_ENV !== 'test'
+    ) {
+      console[console.info ? 'info' : 'log']('Download the Vue Devtools extension for a better development experience:\n'
```
Comment (zoomchan-cxj) [inline packages/hippy-vue/src/runtime/index.js:267]: console.info 一定存在的

File: packages/hippy-vue/src/runtime/index.js [hunk 3]
```diff
@@ -253,4 +254,40 @@ Vue.getApp = getApp;
...
+        + 'https://github.com/vuejs/vue-devtools');
+    }
+  }
+  if (process.env.NODE_ENV !== 'production'
```
Comment (zoomchan-cxj) [inline packages/hippy-vue/src/runtime/index.js:271]: 判断过于冗余

File: packages/hippy-vue/src/runtime/index.js [hunk 4]
```diff
@@ -253,4 +254,40 @@ Vue.getApp = getApp;
...
+  }
+}, 0);
+
+// proxy Vue constructor to add Hippy Vue instance to global.__VUE_ROOT_INSTANCES__
+const ProxyedVue = new Proxy(Vue, {
+  construct(Target, args) {
+    const vm = new Target(...args);
+    if (process.env.NODE_ENV === 'development') {
+      if (!global.__VUE_ROOT_INSTANCES__) global.__VUE_ROOT_INSTANCES__ = [];
+      if (args && args.length && args[0].appName) global.__VUE_ROOT_INSTANCES__.push(vm);
```
Comment (zoomchan-cxj) [inline packages/hippy-vue/src/runtime/index.js:288]: 是不是可以optional chaining

File: packages/hippy-vue/src/runtime/index.js (hunk 1 above)
Comment (shenchaoran) [reply packages/hippy-vue/src/runtime/index.js]: removed

File: packages/hippy-vue/src/runtime/index.js (hunk 4 above)
Comment (shenchaoran) [reply packages/hippy-vue/src/runtime/index.js:268]: js中不支持

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Pull request merged**